README.md

```diff
@@ -4,8 +4,8 @@
 
 _Tradify is a cutting edge investment tool for today's investors._
 
-_Motivated by the recent financial revolution happening in US markets, we wanted to develop a tool that empowers investors of all types, levels, and interest. With Tradify, savvy investors who want to peer into the world of stocks, cryptocurrency, and foreign exchange markets, can access our platform to get up to the minute pricing data and insight in three financial markets in one tool, instead of having to use a separate tool for each.
-This app will run in the browser and feature dynamically updated HTML and CSS powered by jquery, and uses Foundation CDN for styling. Users can retrieve current stocks using [Stock API](https://www.alphavantage.co/documentation/), and cryptocurrency prices using [Coinbase API](https://developers.coinbase.com/)._
+_Motivated by the recent financial revolution happening in US markets, we wanted to develop a tool that empowers investors of all types, levels, and interest. With Tradify, savvy investors who wantpeer into the world of stocks, cryptocurrency, and foreign exchange markets, can access our platform to get up to the minute pricing data and insight in three financial markets in one tool, instead of having to use a separate tool for each.
```

Comment:
```suggestion
_Motivated by the recent financial revolution happening in US markets, we wanted to develop a tool that empowers investors of all types, levels, and interest. With Tradify, savvy investors who want to peer into the world of stocks, cryptocurrency, and foreign exchange markets, can access our platform to get up to the minute pricing data and insight in three financial markets in one tool, instead of having to use a separate tool for each.
```